src/main/js/dashboard-frontend/src/components/details/DetailBody.tsx

```diff
@@ -99,15 +99,15 @@ export default class DetailBody extends Component<
 
           <SpaceBetween size={"l"}>
             <div>
-              <Box margin={{bottom: "xxxs"}} color="text-label">Version</Box>
+              <Box margin={{bottom: "xxxs"}} variant="awsui-key-label"  color="text-label">Version</Box>
               <div>{this.state.service.version}</div>
             </div>
             <div>
               <Popover
                 content='Authorship of the component. "Built-In" Components are provided with Greengrass. "User" components are ones you create.'
                 header={"Component Origin"}
               >
-                Origin
+                <b>Origin</b>
```

Comment:
why change this?

Reply:
Strange, I don't remember changing this. Not needed for sure.